deployments/server/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
Let's avoid adding this or this needs to be able to be disable via ahelm value. This doesn't work if `ServiceMonitor` CRD is not defined.